src/main/java/org/gelecekbilimde/scienceplatform/post/model/entity/CategoryEntity.java

```diff
@@ -32,6 +32,9 @@ public class CategoryEntity extends BaseEntity {
 	@Column(name = "name")
 	private String name;
 
+	@Column(name = "description")
+	private String description;
+
 	@Column(name = "order_number")
 	private Integer order;
```

Comment:
Aşağıdaki kısım için 2 çözüm önerebilirim;

1. Eğer `order` anlamlı geliyorsa veri tabanındaki kolonun adını da `order` yapalım
2. Eğer `order_number` kullanacaksak değişkenin adını da `orderNumber` yapalım.

Benim fikrim 1. seçenekten yana ama 2. seçeneği de yapsak çok da fark etmez, ikisi de anlaşılıyor
```suggestion
	@Column(name = "order")
	private Integer order;
```
```suggestion
	@Column(name = "order_number")
	private Integer orderNumber;
```